doc/Support/Device-Troubleshooting.md

```diff
@@ -4,11 +4,22 @@ source: Support/Device-Troubleshooting.md
 You can use this feature to run Debug on Discovery, Poller, SNMP, Alerts.
 This output information could be helpful for you in troubleshooting a device or when requesting help.
 
-This feautre can be found by going to the deivce that you are troubleshooting in the web ui, clicking on the settings icon menu on far right and selecting 
+This feautre can be found by going to the deivce that you are troubleshooting in the webui, clicking on the settings icon menu on far right and selecting 
 Capture.
 ![Capture-Debug-Icon](/img/capture-debug-icon.png)
 
+### Discovery
+`./discovery.php` will run and output debug infomation.
```

Comment:
Both discovery and poller require more args to work. If this is just indicating what the webui capture system does then I'd just refer to them as Discovery and Poller.

Reply:
@laf. Makes sense, I will edit that.